ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2409,7 +2409,7 @@ periodics:
       - --ignore=redhat-openshift-ecosystem/preflight
       - --ignore=insights-qe/iqe-cost-management-plugin
       - --ignore=ansible/tower-qa
-      - --ignore=rh-openjdk/jdkContainerOcpTests
+      - --ignore=rh-openjdk/jdkContainerOcpTests --ignore=jws-qe/interop-ocp-ci
```

Comment:
please follow the convention and put new options on their own lines

Reply:
Thanks, @smg247, for your review. I fixed the typo in the YAML file.

Reply:
I meant all of the lines, not just this occurrence...